ecs/daemon/outputs/cloudwatch_logs/ecs.conf

```diff
@@ -0,0 +1,19 @@
+[FILTER]
+    Name               aws
+    Match              ecs.*
+    imds_version       v1
```

Comment:
TODO: IMDS version needs to be an env var everywhere.

Reply:
Actually, since we use host mode, its always safe to use V2 with default hop limit of one, so will just use that. No need to worry about this. V2 is always enabled by default now.